independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -0,0 +1,677 @@
+                },
+                "has_more": {
+                  "type": "boolean",
+                  "description": "Whether or not the block has more blocks or not",
+                  "title": "Has more"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search",
+        "description": "Searches all pages and child pages that are shared with the integration. The results may include databases.",
+        "operationId": "Search",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Notion-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "2021-08-16",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "query": {
+                  "type": "string",
+                  "description": "When supplied, limits which pages are returned by comparing the query to the page title.",
+                  "title": "Query"
+                },
+                "sort": {
+                  "type": "object",
+                  "properties": {
+                    "direction": {
+                      "type": "string",
+                      "description": "The direction to sort. Possible values include ascending and descending.",
+                      "title": "Direction"
+                    },
+                    "timestamp": {
+                      "type": "string",
+                      "description": "The name of the timestamp to sort against. Possible values include last_edited_time.",
+                      "title": "Timestamp"
+                    }
+                  },
+                  "description": "sort"
+                }
+              },
+              "required": [
+                "query"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.notion.so/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.notion.so/Privacy-Policy-3468d120cf614d4c9014c09f6adc9091"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity; knowledge management;collaboration"
```

Comment:
@sekharmalla If you read the instructions, it is allowed to add a maximum of two categories. 

![image](https://user-images.githubusercontent.com/56276394/135920078-b38fa3b0-cb38-4bf2-aa48-dcdd57d5bf2d.png)